### DISCUSSION TITLE
对 Pake 在 Windows 上的外观界面两点疑问

### DISCUSSION THREAD

**Discussion by Krrrrrik** [tw93/Pake#158] [Ideas]
Windows 上的标题栏感觉会偏的很大，这个有没有办法能方便的做沉浸😂，我看有些网页的PWA会比较沉浸
![image](https://user-images.githubusercontent.com/84497944/207046960-76c73bff-69ef-4579-9da1-d8cbf8843ef9.png)

以及我发现尺寸调整后是不会被记住的，重新打开的时候都会**重置为默认的尺寸**。

**Comment by Tlntin** [comment]
windows/linux暂时不支持沉浸（其实也算支持，但是支持的不太好，有一些bug），要等上游tauri更新。
尺寸无法记住的问题，dev分支已经解决，等后续合并到主分支后就可以了。
